src/mysql_conn.erl

```diff
@@ -830,3 +830,19 @@ demonitor_processes(List, 0) ->
 demonitor_processes([{_FromPid, MRef}|T], Count) ->
     erlang:demonitor(MRef),
     demonitor_processes(T, Count - 1).
+
+format_status(normal, [_PDict, State]) ->
+	{data, [{"State", State}]};
+format_status(terminate, [_PDict, State=#state{ssl_opts=undefined}]) ->
+	{data, [{"State", State#state{password = hidden}}]};
+format_status(terminate, [_PDict, State=#state{ssl_opts=SSLOpts}]) ->
+	SSLOpts1 = lists:map(
+		fun
+			({cert, _}) -> {cert, hidden};
+			({key, _}) -> {key, hidden};
+			({cacerts, _}) -> {cacerts, hidden};
```

Comment:
Certificates only contain public keys and other non-secret stuff. Only `key` is secret and needs to be hidden IUCC.

Reply:
I just looked at what Ranch hides, TBH ;)